packages/block-editor/src/store/actions.js

```diff
@@ -883,6 +883,18 @@ export function* setNavigationMode( isNavigationMode = true ) {
 	}
 }
 
+/**
+ * Generators that triggers an action used to enable or disable the block moving mode.
+ *
+ * @param {string} isBlockMovingMode Enable/Disable block moving mode.
+ */
+export function* setBlockMovingMode( isBlockMovingMode = '' ) {
```

Comment:
Yeah, I named it after `setNavigationMode` because it mostly works like (and on top of) it. But I needed a way of storing the clientId for the block that initiates moving mode, so I made it take a clientId instead of a boolean. So the naming is awkward, but I'm not sure that `setBlockMovingClientId` is much better, because then we'd end up with `isBlockMovingClientId` which sounds really weird. Am totally open to other suggestions though 😅

Reply:
why not two params then, one for mode `true/false` and one for `clientId`?